basic_cmds.go

```diff
@@ -68,3 +68,34 @@ func (c *Client) Whoami() (*ConnectionInfo, error) {
 
 	return i, nil
 }
+
+// Clientupdate sets a property of the client to a given value.
```

Comment:
Could we just make this ClientUpdate please @irgendwr

I would also be tempted to make this take map[string]interface{} or a variadic interface{}; with that done adding constants for ClientNickname, ClientIsTalker etc it would be easy to use clientupdate as documented to set multiple parameters at the same time.

Reply:
How about a variadic CmdArg since it can be passed straight to WithArgs? (`properties ...CmdArg`)
Or would you prefer `properties map[string]interface{}`?

Reply:
Sounds like a good idea.